TypeCobol/Compiler/Sql/CodeElements/Statements/ExecuteImmediateStatement.cs

```diff
@@ -0,0 +1,21 @@
+﻿using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Sql.Model;
+
+namespace TypeCobol.Compiler.Sql.CodeElements.Statements
+{
+    public class ExecuteImmediateStatement : SqlStatementElement
```

Comment:
As I've already commented, you have to add the EXECUTE keyword in the statement-starting keywords list